### ISSUE TITLE
[Web] [Accessibility] [11yMAS][HighImpact][Keyboard Navigation][MAS2.1.1] [Interactive Controls in Report Generation frame are not accessible by Keyboard.]

### ISSUE THREAD

**Issue by DasaradhaKamjula** [OfficeDev/script-lab-2017#682]
**User Experience:**
User who rely on keyboard will not be able to access those controls if they are not accessible by keyboard which may lead for external help.

**Test Environment:**
OS Build: 16299.19
Browser: Microsoft Edge 41.16299.15.0
**Attachments:**
[MAS2.1.1_Report generation controls are not accessible.zip](https://github.com/OfficeDev/script-lab/files/1440688/MAS2.1.1_Report.generation.controls.are.not.accessible.zip)
 
**Repro Steps:**

1. Open the URL: http://scriptlabdocsdemo.azurewebsites.net/Office%20Dev%20Center%20-%20Docs%20-%20Build%20your%20first%20Excel%20add-in.html .
2. Navigate to “Report Generation” Frame using the Tab key.
3. Focus will be inside the script that is editing.
4. Verify whether user is able to access all the Interactive controls in the Report generation Frame . 

**Current Behavior:** 
Interactive Controls in Report Generation frame are not accessible by Keyboard.

**Expected Behavior:**
All Interactive Controls in Report Generation frame should be accessible by Keyboard. So that keyboard user can be able to access all controls without any problem.
 
**Note:**
1. Same issue is repro for the [MAC + safari][ Build info: OS Build: Mac OS Sierra 0.12.6 Browser: Safari 11.0.1] Interactive Controls in the Report generation frame is not accessible by the keyboard.
2. All above issues are repro for the Entry Point Page.

**MAS Reference:**
https://microsoft.sharepoint.com/%3Aw%3A/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7B8492c4eb-c179-40ae-8777-cd044ed725a2%7D

**Suggested Fix:**
https://microsoft.sharepoint.com/teams/msenable/mas/pages/mas-details.aspx?masid=2.1.1
